kg_microbe/transform_utils/madin_etal/madin_etal.py

```diff
@@ -222,19 +220,34 @@ def run(self, data_file: Union[Optional[Path], Optional[str]] = None, show_statu
                     tax_name = filtered_row[ORG_NAME_COLUMN]
                     tax_node = [tax_id, NCBI_CATEGORY, tax_name]
 
-                    metabolism = metabolism_map.get(filtered_row[METABOLISM_COLUMN], None)
+                    # block handling "metabolism" column from Madin etal dataset/CSV sheet
+                    metabolism = self.madin_metpo_mappings.get(
+                        filtered_row[METABOLISM_COLUMN], None
+                    )
                     if metabolism:
+                        # create metabolism node and edge to tax_id
+                        # use `biolink equivalent` for 'category' and
+                        # for 'predicate', `biolink equivalent` from properties
+                        category = metabolism.get("biolink_equivalent", METABOLISM_CATEGORY)
+                        predicate_biolink = metabolism.get("predicate_biolink_equivalent", "")
+                        # fallback: if no biolink equivalent use `biolink:has_phenotype`
+                        if predicate_biolink:
+                            predicate = predicate_biolink
+                        else:
+                            predicate = "biolink:has_phenotype"
                         metabolism_node = [
-                            metabolism['curie'],
-                            METABOLISM_CATEGORY,
-                            metabolism['label'],
+                            metabolism["curie"],
+                            category,
```

Comment:
@sujaypatil96 I think before this was the biolink node category -- is the new category from the METPO sheet? Maybe its the METPO parent?

Reply:
@realmarcin yes previously, `METABOLISM_CATEGORY` for example had a hardcoded value of `biolink:ActivityAndBehavior`, but now the _category_ is being inferred from the METPO sheet using the `biolink equivalent` column.